src/server/scripts/BrokenIsles/VaultOfTheWardens/boss_tirathon_saltheril.cpp

```diff
@@ -0,0 +1,554 @@
+    SPELL_DARKSTRIKES = 191941,
+    SPELL_FEL_MORTAR = 202913,
+    SPELL_FEL_MORTAR_TRIGGER = 202920,
+
+
+    // 14:56:56.852
+    //191774 knockback effect casted by player so basically player knocking himself?
+    SPELL_SWOOP_KNOCK_BACK = 191774, //Casted by a player knockback?
+    SPELL_SWOOP_DUMMY = 191765, //dummy
+    SPELL_SWOOP_JUMP = 191766, //Jump
+    SPELL_SWOOP_DAMAGE = 191767,//Damage & knockback everyone in path from jump to destination.
+
+    //Aura Update 14:56:53.585
+    //ID - 191999 Dark Energies
+    SPELL_DARK_ENERGIES = 191999,
+
+    // 14:56:56.853 Number: 39408
+    // ID - 211057 Throw Glaive
+    // Swoop 191765 triggering
+    // 14:56:56.877 Number: 39412
+    SPELL_THROW_GLAIVE_MAIN_HAND = 211057,
+    SPELL_THROW_GLAIVE_OFF_HAND = 211058,
+
+    // ID - 192108 Glaive Damage
+    // 14:56:56.964 Number: 39424 again swoop
+    SPELL_GLAIVE = 192108, 
+
+    //ID - 211068 Recall Glaive
+    //Time : 14:57:01.983
+    SPELL_RECALL_GLAIVE = 211068,
+
+    // ID - 191823 Furious Blast
+    // 14:56:56.991 Number: 39436
+    SPELL_FURIOUS_BLAST = 191823,
+
+    //ID - 191853 Furious Flames
+    //14:57:00.983
+    SPELL_FURIOUS_FLAMES = 191853,
+
+    //14:57:07.781
+    //ID - 192504 Metamorphosis
+    SPELL_METAMORPHOSIS = 192504,
+
+    // ID - 212558 Summon Fel Fury
+    //4264.336 Y: -431.36285 Z: 259.4851 location
+    //4267.597 Y: -433.0495 Z: 259.57462
+    // X: 4284.3716 Y: -441.7253 Z: 259.60638 second?>
+    /*
+        [0] Points: X: 4244.3003 Y: -421.0004 Z: 259.53052
+        [1] Points: X: 4264.336 Y: -431.36285 Z: 259.56845
+        [2] Points: X: 4284.3716 Y: -441.7253 Z: 259.60638
+        [3] Points: X: 4284.3716 Y: -441.7253 Z: 259.60638
+    */
+    SPELL_SUMMON_FELL_FURY = 212558,
+    SPELL_SCORCH = 212541,
+    SPELL_SELF_DESTRUCTION = 212551,
+    SPELL_FEL_EMISSION = 212547,
+
+    // ID - 202780 Summon Laser Beam Target
+    SPELL_LASER_BEAM = 202780,
+
+    //14:57:25.567 
+    //ID - 190830 Hatred First Target Himself ?
+    //ID - 190833 Hatred Target Summuned NPC?
+    //Entry: 96241
+    SPELL_HATRED = 190830, //SELF TARGET
+    SPELL_HATRED_TARGET = 190833, //UNIT LASER?
+    SPELL_HATRED_DAMAGE = 190836, //Damage if u get in AT
+    //SpellID: 202780 (Summon Laser Beam Target)
+    SPELL_SUMMON_LASER_BEAM_TARGET = 202780,
+    //BONUS ROLL 226649
+};
+enum Talk
+{
+    TALK_ENTER_COMBAT = 0,
+    TALK_EMPOWER_DARKSTRIKE = 1,
+    TALK_EMPOWER_DARKSTRIKE_HIT = 2,
+    //icon + Text: I will cast you all aside...
+    TALK_FURIOUS_BLAST_TO_PLAYER = 3,
+    TALK_FURIOUS_BLAST = 4,
+    //Text: Anger, hate, rage! These will fuel me with the power I crave!
+    TALK_METAMORPHOSIS = 6,
+    TALK_DEATH = 8,
+};
+enum Data
+{
+    DATA_JUMP = 1
+};
+
+enum Actions
+{
+    ACTION_METAMORPHOSIS = 1,
+    ACTION_SUMMON_FEL_FURY = 2,
+};
+const Position jumpPos = { 4284.973f, -451.2752f, 259.66946f, 3.123f };
+const Position spawnFury[4] = {
```

Comment:
I would honestly move these const position lines to the instance_*.h file, looks cleaner imo.